x-pack/plugins/apm/public/components/alerting/alerting_flyout/index.tsx

```diff
@@ -55,7 +55,7 @@ export function AlertingFlyout(props: Props) {
       services.triggersActionsUi.getAddAlertFlyout({
         consumer: APM_SERVER_FEATURE_ID,
         onClose: onCloseAddFlyout,
-        alertTypeId: alertType,
+        ruleTypeId: alertType,
```

Comment:
@XavierM I guess you left `alertType` here for a later refactoring on o11y side, right?

Reply:
Yes, I was just trying to take care of the `triggers_actions_ui` plugin.